src/scripts/addSuperUser.py

```diff
@@ -0,0 +1,45 @@
+import os
+from sqlalchemy import create_engine, MetaData, insert, select
+from sqlalchemy.sql import func
+
+engine = create_engine(os.getenv("DATABASE_URL"))
+connection = engine.connect()
+metadata = MetaData()
+metadata.reflect(engine)
+
+
+def createSuperUser(login, password):
+
+    # create hasched password
+    # I think we might get the pwd_context from security.py !
+    # problems with imports, thus temporary
```

Comment:
I think it would be better to call the code that creates a user directly rather than reproducing this process in the script.

Reply:
(And i wonder if we need to create a script or just embed this super usercreation in the API deployment process. See my comment in  #26 )